numpy/distutils/misc_util.py

```diff
@@ -2371,9 +2352,8 @@ def show():
                 """
                 Show libraries in the system on which NumPy was built.
 
-                Print information about various resources (libraries, library
-                directories, include directories, etc.) in the system on which
-                NumPy was built.
+                Print information about various resources in the system
+                on which NumPy was built.
```

Comment:
The information presented no longer is the build-time information.
```suggestion
                including available intrinsic support and BLAS/LAPACK library
                in use
```

Reply:
Done via [49ed995](https://github.com/numpy/numpy/pull/21468/commits/49ed9952bffcf55fa4d298e05aee3d18138fe2a3)